### ISSUE TITLE
Filters are missing from the TypeScript types of v1.1.7

### ISSUE THREAD

**Issue by zaslavskyv** [peerigon/angular-expressions#36]
```
// TS2305: Module '"angular-expressions"' has no exported member 'filters'.
import { filters } from "angular-expressions";
filters.uppercase = (input: string): string => input.toUppercase();
```

**Comment by edi9999** [comment]
Thanks for the issue, fixed in version 1.1.8 !

**Comment by zaslavskyv** [comment]
Cheers!
